metadata.md

```diff
@@ -3,15 +3,25 @@ layout: default
 css_id: metadata
 ---
 
-# Metadata
+# Roles and Metadata
 
-Metadata files provide information that clients can use to make update decisions. Different metadata files provide different information. The various metadata files are signed by different keys as are indicated by the root role. The concept of roles allows TUF to only trust information that a role is trusted to provide.
+TUF uses metadata to create verifiable records about the state
+of a repository or application at a specified time and, as such, clients can use
+it to make decisions on which files to update. Different metadata files provide
+different information, and will be signed by different roles.
 
-The signed metadata files always include the time they were created and their expiration dates. This ensures that outdated metadata will be detected and that clients can refuse to accept metadata older than that which they've already seen.
```

Comment:
> The signed metadata files always include **the time they were created** and their expiration dates.

Although once true, the time that metadata is created is no longer included in metadata.  We should instead say that, "The signed metadata files always include their expiration dates.

We should verify that the creation timestamp is not mentioned elsewhere in this document.

Reply:
fixed.

Reply:
I'll check for other references to the creation timestamp as I go forward.